who_is_active.sql

```diff
@@ -2740,7 +2741,15 @@ BEGIN;
 													END +
 												N'')''
 											WHEN y.wait_type = N''CXPACKET'' THEN
-												N'':'' + SUBSTRING(y.resource_description, CHARINDEX(N''nodeId'', y.resource_description) + 7, 4)
+												N'':'' + SUBSTRING(y.resource_description, 
+	                                                               CHARINDEX(N''nodeId'', y.resource_description) + 7, 

```

Comment:
Did you test this on any pre-2016 versions by any chance? Just want to make sure it doesn't break back-compat. Also, spaces. Please switch to tabs, and then I'll flip the entire script over to spaces, because I've become enlightened of late.

Reply:
Yessir. Used to doing that for working on other stuff.
![image](https://user-images.githubusercontent.com/2136037/97510450-22b5f300-195b-11eb-9bf7-4d91e6721413.png)